### PR TITLE
spellchecking openwoo.app

### PR DESCRIPTION
**Pull request by MWest2020**

### CODE REVIEW

File: docs/GOVERNANCE.md [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
+Het succes en de continue verbetering van het OpenWoo.app project zijn afhankelijk van een effectieve governance-structuur. Een belangrijk onderdeel van deze structuur is de stuurgroep, die verantwoordelijk is voor het beoordelen en prioriteren van de issues die op GitHub worden ingediend, en voor het bepalen welke daarvan worden opgenomen in de roadmap van het project.
 
 ## Stuurgroep
 
 De Stuurgroep is samengesteld uit verschillende leden die een breed scala aan perspectieven en expertise inbrengen in het project. De huidige deelnemende leden van de stuurgroep zijn:
 
 - **Conduction (Leverancier)**: Met een rijke ervaring in het leveren van technologische oplossingen, brengt Conduction waardevolle inzichten en technische expertise in het project.
 - **xxlnce (Leverancier)**: Als leverancier draagt xxlnce bij aan de technologische vooruitgang en zorgt ervoor dat het project blijft voldoen aan de behoeften van de gebruikers.
-- **Epe (Pilot Gemeente)**
```
Comment (remko48) [inline docs/GOVERNANCE.md:11]: Accidental change?

File: docs/Kosten.md [hunk 2]
```diff
@@ -11,7 +11,7 @@ Onderstaande treft u de bij ons bekende kosten per component die leveranciers in
 | [Conduction](https://conduction.nl/)           | [Ruben van der Linde (info@conduction.nl)](mailto:info@conduction.nl)                |
 | [xxllnc](https://xxllnc.nl/)                   | [Arthur van Ulden (sales.zaakgericht@xxllnc.nl)](mailto:sales.zaakgericht@xxllnc.nl) |
 | [Sim Groep](https://www.simgroep.nl/)          | [Twan Meijerink(t.meijerink@simgroep.nl)](mailto:t.meijerink@simgroep.nl)                                   |
-| [OpenGemeenten](https://www.opengemeenten.nl/) | [Micheline Tasseron (micheline@opengemeenten.nl)](mailto:micheline@opengemeenten.nl) |
+| [OpenGemeenten](https://www.opengemeenten.nl/) | [Jan-Thijs Oosterhoudt (hallo@opengemeenten.nl)](mailto:hallo@opengemeenten.nl) |
```
Comment (remko48) [inline docs/Kosten.md:14]: Is this the correct email address?

**Review by remko48 — APPROVED** [review]
Approved with two small comments, please resolve those before merging.

File: docs/GOVERNANCE.md (hunk 1 above)
Comment (MWest2020) [reply docs/GOVERNANCE.md]: nice one

**Pull request merged**